core/parser.php

```diff
@@ -66,7 +66,7 @@ public function replace_mentions_for_display($text)
 			$start_text = $text;
 		}
 
-		for($i; $i < 100; $i++)
+		for ($i; $i < 100; $i++)
```

Comment:
@s9e This code will be gone in any beta version anyway, it was just quick'n'dirty coded for time measuring and I kept it for testing purposes.
But you are right indeed. I'll keep that until I will remove that code.